### PR TITLE
Aserto.DSLoad.v0.30.6

### PR DESCRIPTION
**Pull request by gertd**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/150869)

### CODE REVIEW

**Comment by 2505lis** [comment]
https://sites.google.com/view/ano-center-gemini25

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   317F5FF61D16CBD0C492012C06CA0B3B5D0B514EE143592B4071CFD0D92DCCAF
 Expected hash: eef80398baeff0b82f5beea8a6f896e2a82851cced5cb812a2c4b732dcee00c3
 
 4/24/2024 11:05:00 PM
 
 ****************************************************************************************
 Installer Validation
 ****************************************************************************************
 

(Automated response - build 877.)

File: manifests/a/Aserto/DSLoad/0.30.6/Aserto.DSLoad.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageIdentifier: Aserto.DSLoad
+PackageVersion: 0.30.6
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ds-load-auth0.exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/aserto-dev/ds-load/releases/download/v0.30.6/ds-load_0.30.6_windows_x86_64.zip
+  InstallerSha256: EEF80398BAEFF0B82F5BEEA8A6F896E2A82851CCED5CB812A2C4B732DCEE00C3
```
Comment (stephengillie) [inline manifests/a/Aserto/DSLoad/0.30.6/Aserto.DSLoad.installer.yaml:13]:
```suggestion

```

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Aserto.DSLoad__DefaultSource\ds-load-auth0.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Aserto.DSLoad__DefaultSource\ds-load-azuread.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Aserto.DSLoad__DefaultSource\ds-load-cognito.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Aserto.DSLoad__DefaultSource\ds-load-google.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Aserto.DSLoad__DefaultSource\ds-load-ldap.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Aserto.DSLoad__DefaultSource\ds-load-okta.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Aserto.DSLoad__DefaultSource\ds-load.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ds-load-auth0.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ds-load-azuread.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ds-load-cognito.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ds-load-google.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ds-load-ldap.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ds-load-okta.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ds-load.exe returned exit code: 1

(Automated response - build 877.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**